app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
 <resources>
+    <!-- LOCALIZATION NOTE (app_name): The string 'app_name' will be used any time the application
+         name is to be displayed to the user inside of the application as well as any system launcher
+         such as the text bellow the application launch icon. -->
     <string name="app_name">Firefox Reality</string>
-    <string name="new_tab">New tab</string>
-    <string name="menu_enter_private_browsing">Enter Private Browsing</string>
-    <string name="menu_exit_private_browsing">Exit Private Browsing</string>
-    <string name="menu_focus_mode">Focus Mode</string>
+    <!-- LOCALIZATION NOTE (keyboard_enter_label): The string 'keyboard_enter_label' is used on the
+         virtual keyboard to label the enter key. -->
     <string name="keyboard_enter_label">ENTER</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_camera): '%1$s' will be replaced at run time with the app name.
```

Comment:
could we replace `%1$s` with `{app}` or `{appName}`?

Reply:
[`the application name` is clear above](https://github.com/MozillaReality/FirefoxReality/pull/626/files#diff-01dafb3fd0217441330103cfc8300094R2), but here and elsewhere `the app name` is ambiguous (to me, at least)

Reply:
For now, we'll only support %1$S in the l10n tooling, let's stick to that, please.